src/helper/util.ts

```diff
@@ -266,15 +266,15 @@ export class ZkWasmUtil {
     message += params.avator_url;
     message += params.circuit_size;
 
-    // Additional params afterwards
-    if (params.initial_context) {
-      message += params.initial_context_md5;
-    }
-
     message += params.prove_payment_src;
     for (const chainId of params.auto_submit_network_ids) {
       message += chainId;
     }
+
+    // Additional params afterwards
+    if (params.initial_context) {
```

Comment:
This change will not affect our verification , correct?
Any issue if we do not put it at last? (I agree as it is optional so it should be put at last), just want to understand the affect

Reply:
This is a essential fix as currently we put context after auto_submit_network_ids in UI so we should keep this rule